book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+  You may need to develop a Glossary specific to your event and area, or consider what Glossaries have been developed by research funding bodies or University Communications teams and provide links if applicable. 
+- Finally, **Robustness**.
+  Robustness refers to the importance of content on your event page's website being compatible with different browsers, now and in the future,
+  and how compatible the content is with different technologies such as screen readers and other assistive technologies. 
+
+[Mozilla's Developer Documentation on Web Accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG) provides a good overview of 
+how to meet these 4 web accessibility standards and also provides more examples of how to meet each criterion. 
+If you are interested in learning more, you can also review the full guidance published by the [Web Accessibility Initiative](https://www.w3.org/WAI/standards-guidelines/wcag/).
+
+
+(cl-inclusive-events-comms-website-content)=
+### Website Content 
+Depending on your event and where you host online information, you Ensure that your event information covers:
+- How to attend (joining links for online content and location information for in-person events).
+- When and where your event is.
+- What accessibility support there will be for your event
+- Who the event hosts are and who the session leads will be. This can help attendees know what to expect. 
+- Information about what your event is and what is expected from the audience. This includes if your attendees are expected to have background information on a topic and if your event expects active participation from the attendees. 
+- The program or agenda for the event including planned start and end times and scheduled breaks.
+- The event's Code of Conduct and Privacy notice.
+- Documentation or information about the honorarium, reimbursement, or renumeration policy for your event. 
+- Information for a point of contact to get in touch with if there are any issues with joining or if attendees want follow-up information about the event.
+
+Consider also providing any content that is on a website in an offline format such as an information booklet or flyer. 
+This can help attendees access important event information without requiring internet access and can also help attendees with screen readers or who prefer to read information offline. 
+HTML and plain text is often the most accessible format, and some PDF software provides several accessibility features such as Adobe Reader which can be accessed or downloaded for free. 
+
+(cl-inclusive-events-registration)=
+## Registration
+If you are using forms or an external registration software, ensure that the registration forms and platform are accessible for all attendees. 
+This includes ensuring that if a time limit is imposed, it can be changed or turned off as mentioned above in the event website section. 
```

Comment:
I'm not sure what time limit you mean, the one of e.g. eventbrite for "15 mins before your ticket is put back"?

Reply:
I updated this section - it would be great if you could review the changes as I may now be over-explaining things and giving too many examples.